### DISCUSSION TITLE
Artillery Load Testing: How to Capture, Store, Log 'guid' from Received Response and Pass it in Another Request?

### DISCUSSION THREAD

**Discussion by newuserbk** [artilleryio/artillery#2363] [Artillery]
When executing an Artillery test via the Windows command line, I observe the following response

In my YML configuration file, I've implemented code to capture and log specific fields, but unfortunately, I am unable to see the captured fields and logs (Also I am not able to Pass value from one request to another. e.g. in My case case I have to pass guid to next request). Any insights into what might be missing or causing this issue would be greatly appreciated. Thanks in advance for your assistance

**Comment by InesNi** [comment]
Hey @newuserbk 👋🏻 !

The `capture` field needs to be nested inside one of the method keys (e.g. inside `post`), but you've put it at the same level as them

Here is the information about the syntax: https://www.artillery.io/docs/reference/engines/http#syntax

Hope that helps 🚀

**Reply by newuserbk** [reply]
@InesNi 
 my first request response received but I am not able to check what happened with my second request whether requestId captured from first request is being passed to second request or not

I have provided my complete YML file. Could you please help me with the syntax.
Because with my current YML there is no error but not able to verify second request
